README.md

```diff
@@ -32,27 +32,24 @@ The web form will then retrieve back the image using the *setCurrentProfileImage
 
    #### Depending on the Cloud Object Storage Package
 
-   The COS Image Upload template relies on the Cloud Object Storage package. In order for the template to function properly you must configure it to use the service credentials of an existing Cloud Object Storage instance. IMPORTANT NOTICE: Your Cloud Object Storage instance must contain credentials with the HMAC keys present(see below) and at least one Bucket present. 
+   The Upload Image template relies on the Cloud Object Storage package. In order for the template to function properly      you must configure it to use the service credentials of an existing Cloud Object Storage instance. IMPORTANT NOTICE: Your      Cloud Object Storage instance must contain credentials with the HMAC keys present(see below) and at least one Bucket          present.
    There are three approaches you can take:
-    * **Create an new instance**: Selecting this option will take you to the IBM Cloud page for creating Cloud Object Storage instances. It is important that after you create your instance you create a set of Service Credentials that contain the needed HMAC keys (See below)
-    * **Input your own credentials**: Selecting this will prompt you to manually enter your own credentials for a COS instance
-    * **Existing Instances**: If you already have any COS instances created they should be automatically populated in the dropdown. Clicking an existing instance will attempt to fetch the credentials as well as any Buckets existing on that instance
-    * **Important Note**
-    In order for the COS Template to be deployed properly your COS instance should have HMAC keys present as well as an already existing bucket.  For information on creating HMAC keys refer to this documentation: [Create COS Service Credentials](https://console.bluemix.net/docs/services/cloud-object-storage/iam/service-credentials.html#service-credentials)
+   
+   * **Create an new instance**: Selecting this option will take you to the IBM Cloud page for creating Cloud Object Storage instances. It is important that after you create your instance you create a set of Service Credentials that contain the needed HMAC keys (See below)
+   * **Input your own credentials**: Selecting this will prompt you to manually enter your own credentials for a COS instance
+   * **Existing Instances**: If you already have any COS instances created they should be automatically populated in the dropdown. Clicking an existing instance will attempt to fetch the credentials as well as any Buckets existing on that instance
+   ****Important Note****
+   In order for the COS Template to be deployed properly your COS instance should have HMAC keys present as well as an already existing bucket.  For information on creating HMAC keys refer to this documentation: [Create COS Service Credentials](https://console.bluemix.net/docs/services/cloud-object-storage/iam/service-credentials.html#service-credentials)
```

Comment:
"For the COS Template to be deployed properly, your COS instance should have..."  For vs in order for & a comma after properly.